query-service-impl/src/main/java/org/hypertrace/core/query/service/ExecutionContext.java

```diff
@@ -173,4 +254,17 @@ public ResultSetMetadata getResultSetMetadata() {
   public LinkedHashSet<Expression> getAllSelections() {
     return this.allSelections;
   }
+
+  public Optional<Double> getTimeSeriesColumn(String columnName) {
+    if (this.timeSeriesColumnMap.containsKey(columnName)) {
+      return Optional.of(Double.parseDouble(this.timeSeriesColumnMap.get(columnName)));
```

Comment:
Curious: Why do we need to convert the period to double?
Can we rename the function name to something that represents the period?
e.g `getPeriodForTimeSeriesColumn`

Reply:
Since we are dividing, double will give more precision. Renamed accordingly.